src/NuGet.Services.EndToEnd/AutoCompleteResultTests.cs

```diff
@@ -103,5 +103,44 @@ public async Task V2PackageIdsAutocompleteResultsMatchV3Results(
                 Assert.Equal(v2Response, v3Response.Data);
             }
         }
+
+        [GalleryTestTheory]
+        [InlineData(PackageType.SemVer1Stable, true, null)]
+        [InlineData(PackageType.SemVer1Stable, false, null)]
+        public async Task UnicodePackageIdV2PackageVersionsAutocompleteResultsMatchV3Results(
+            PackageType packageType,
+            bool includePrerelease,
+            string semVerLevel)
+        {
+            // Arrange
+            var package = await _pushedPackages.PrepareAsync(packageType, _logger, unicodeId : true);
```

Comment:
I don't see where this new parameter is used.

Reply:
Sorry, I didn't stage one more file change.